README.md

```diff
@@ -42,7 +42,7 @@ on the container. This is done to allow for FPM to run on the container
 alongside the web server. The `procs.yml` file it runs with contains the
 commands and arguments for both `php-fpm` and the web-server.
 
-When the buildpack runs, you will see in the logs what processes are addded to
+When the buildpack runs, you will see in the logs what processes are added to
```

Comment:
Are you planning to add anything here about the live reload?

Reply:
Good call, pushed a change to README